### DISCUSSION TITLE
Qt client, a problem with teamtalk's peformence

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1584] [Q&A]
### Description

so guys, i'm sorry that i don't have any exact reporting of what's happening, but i will tell you because it's something general: so if you compare classic client with Qt, you will understand my point: i feel that classic client is much more faster than the Qt client: for example, if i'm pressing a button to open a dialogue, in classic client, it will do it as soon as i do it actually, but in Qt client, sometimes it has really small little tiny little delay or lag to do it: for example, try it with pressing enter on someone's user to open the privat message popup, or try with pressing controle + u on a user to open the volume popup, and compare it with the classic client: i don't think that this problem is because of my windows vertion, because i'm windows 10, and one of my friends also have this problem with her windows 11, so that's y i reported this to you, please fix this  and make it faster, since you are gonna aband the classic client and Qt client will be the main one.: my windows vertion: Windows 10 21H1 (x64) build 19043.1826 Qt client vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by ChrisDuffley** [comment]
i'm also having performance issues, especially when tabbing around the window and also the tree view. I've heard multiple people have this problem as well, @dglee42 first reported this when loading the Update Channel dialog, for instance. Also, after downgrading to 5.10 to see if that would resolve the issue, it did not.

Any thoughts on this?

**Comment by amirmahdifard** [comment]
yeah, it has this problem everywhere, thats y i didn't say only this dialogue is slo. this problem should fix really in Qt. the classic client is really fast

**Comment by amirmahdifard** [comment]
@ChrisDuffley as you sayd about downgrate to 5.10, i downgrated my tt to 5.8.0, it stil have this problem.

**Comment by amirmahdifard** [comment]
i think that if they fix these miner bugs, people will have no more  reezen to use classic client. i feel that if these bugs gets fix, the qt client is more exciting than the classic.
